languages/tree-sitter-stack-graphs-java/test/imports_and_exports/import_method.java

```diff
@@ -0,0 +1,17 @@
+/*--- path: Importer.java ---*/
+import foo.Foo;
+
+public class Importer {
+  public static void main(String[] args) {
+    Foo.bar();
+     // ^ defined: 15
+
+  }
+}
+
+/* --- path: foo/Foo.java ---*/
```

Comment:
@hendrikvanantwerpen With the path set the way it is, this is valid Java without the package listed at the top. @robrix and I tested it. We plan to add in support for explicit packages soon. Are you okay with leaving this as-is since it is technically valid?